apps/family-public-law/fpl-cron-ccd-data-migration-tool/perftest/01.yaml

```diff
@@ -6,5 +6,7 @@ spec:
   releaseName: fpl-cron-ccd-data-migration-tool
   values:
     job:
-      image: hmctspublic.azurecr.io/fpl/cron-ccd-data-migration-tool:pr-101-cc746c7-20230317112724 #{"$imagepolicy": "flux-system:perftest-fpl-cron-ccd-data-migration-tool"}
-      schedule: "30 12 30 2 *"
\ No newline at end of file
+      image: hmctspublic.azurecr.io/fpl/cron-ccd-data-migration-tool:pr-102-1511ef2-20230327093303 #{"$imagepolicy": "flux-system:perftest-fpl-cron-ccd-data-migration-tool"}
+      schedule: "30 12 30 2 *"
+      environment:
+        ENABLED: false
```

Comment:
does this cluster need a migration ID?

Reply:
migration id is set on the base file - fpl-cron-ccd-data-migration-tool.yaml